docs/animated.md

```diff
@@ -311,7 +315,7 @@ Config is an object that may have the following options:
 - `velocity`: Initial velocity. Required.
 - `deceleration`: Rate of decay. Default 0.997.
 - `isInteraction`: Whether or not this animation creates an "interaction handle" on the `InteractionManager`. Default true.
-- `useNativeDriver`: Uses the native driver when true. Default false.
+- `useNativeDriver`: Uses the native driver when true. Required.
```

Comment:
If it will issue a warning saying it is required, the type system enforcing it is present seems good to me. My guess is that there were too many existing usages to do the same in Flow.